drain-service/drain-service.yaml

```diff
@@ -1,36 +0,0 @@
-apiVersion: apps/v1
```

Comment:
just a suggestion maybe we can keep the yaml file so it's easier to dev on this service?

Reply:
nah we can use https://github.com/sanjay920/opni-manual-install
main reason is we want everyone else to use opnictl